### PR TITLE
Add border to profile picture & style chat icon

### LINKED ISSUE #4419: Default avatars for users and group chats
*Type*: Bug/Feature

Default avatars for users and group chats

Zeplin:
Web: https://zpl.io/be4G4nm 
App: zpl://screen?sid=5b0547e0062822812e56f027&pid=5a55d587ba304eed4d2c942f

avatars

<blockquote><div> Zeplin</div><div><strong><a href="https://zeplin.io/">Zeplin - Login</a></strong></div><div>Collaboration app for UI designers and frontend developers</div></blockquote>

### PR DESCRIPTION
**Pull request by cammellos**
Fixes #4419 

### Summary:

* Added a transparent border to profile images
* Allowed multiple colors in chat-icons
* Changed `identicon` to display smaller icons

### Review notes (optional):

I have consolidated most of the logic for creating pictures in a single place  `status-im.chat.views.photos`.

The method for adding a border is to superimpose a `div` with a invisible border, of all the method considered that is the simplest, afaics.

### Testing notes:

* Previously generated images will not change.
* I have tried to find all the profile pics in the app, but without a comprehensive list I might have missed some.

status: ready

### CODE REVIEW

File: src/status_im/ui/components/styles.cljs [hunk 1]
```diff
@@ -52,6 +52,14 @@
...
 (def separator-color "#0000001f")
 (def default-chat-color color-purple)
+(def chat-colors ["#fa6565"
+                  "#7cda00"
+                  "#887af9"
+                  "#51d0f0"
+                  "#fe8f59"
+                  "#d37ef4"])
+(defn random-chat-color []
+  (rand-nth chat-colors))
```
Comment (goranjovic) [inline src/status_im/ui/components/styles.cljs]: Maybe better place this in `status-im.ui.components.colors`

**Review by goranjovic — APPROVED** [review]

File: src/status_im/chat/styles/photos.cljs [hunk 2]
```diff
@@ -1,8 +1,24 @@
 (ns status-im.chat.styles.photos)
 
-(def photo-size 36)
+(def default-size 36)
+(def border-color "#ccd3d6")
```
Comment (jeluard) [inline src/status_im/chat/styles/photos.cljs]: Is that a new color? Can it be deifined in `colors` ns?

File: src/status_im/chat/views/photos.cljs [hunk 3]
```diff
@@ -6,18 +6,26 @@
...
 (defview member-photo [from]
   (letsubs [photo-path [:get-photo-path from]]
-    (photo from  (if (string/blank? photo-path)
-                   (identicon/identicon from)
-                   photo-path))))
+    (photo (if (string/blank? photo-path)
+             (identicon/identicon from)
+             photo-path)
+           {:accessibility-label :member-photo
+            :size style/default-size})))
```
Comment (jeluard) [inline src/status_im/chat/views/photos.cljs]: Not aligned :)

File: src/status_im/ui/components/chat_icon/styles.cljs [hunk 4]
```diff
@@ -36,10 +36,10 @@
           :border-radius 32}))
 
 (def default-chat-icon-text
-  {:margin-top  -2
-   :color       color-white
-   :font-size   16
-   :line-height 20})
+  {:color       color-white
```
Comment (jeluard) [inline src/status_im/ui/components/chat_icon/styles.cljs]: Can use colors ns

File: src/status_im/ui/components/styles.cljs [hunk 5]
```diff
@@ -52,6 +52,14 @@
 (def chat-background color-light-gray)
 (def separator-color "#0000001f")
 (def default-chat-color color-purple)
+(def chat-colors ["#fa6565"
```
Comment (jeluard) [inline src/status_im/ui/components/styles.cljs]: Does it make sense to have those in colors ns?

**Review by jeluard — CHANGES_REQUESTED** [review]

**Comment by cammellos** [comment]
@jeluard updated!

**Review by jeluard — APPROVED** [review]

**Comment by cammellos** [comment]
Looking at the failed e2e tests, I pushed a change to re-introduce `:chat-id` label, might ask you to restart them if not automatic.

**Comment by lukaszfryc** [comment]
@cammellos all looks good. One small thing - maybe you could check if the character is really centered inside the avatar. If it is, please merge.

**Comment by cammellos** [comment]
@lukaszfryc as far as I can see from the code, it should be center aligned, `justify-content :center`

**Pull request merged**